src/EtherFiRewardsRouter.sol

```diff
@@ -40,7 +39,7 @@ contract EtherFiRewardsRouter is OwnableUpgradeable, UUPSUpgradeable  {
     }
 
     function withdrawToLiquidityPool() external {
-        if (!roleRegistry.hasRole(ETHERFI_ROUTER_ADMIN, msg.sender)) revert IncorrectRole();
+        if (msg.sender != admin) revert IncorrectRole();
```

Comment:
@vvalecha519 I think we can make this function `withdrawToLiquidityPool` permission-less